ncm-spma/src/main/perl/spma/dnf.pm

```diff
@@ -534,19 +534,26 @@ sub Configure
     $self->info("Will skip removal of running kernel: $kvers");
```

Comment:
Not sure you need this info() log line any more since the real work is done later (and also logged via info()?) It records the value of $kvers but not sure that's useful enough for info.

Reply:
was debating about this myself. I agree, don't think it's much of a useful info.